client/src/pages/JoinRoomPage/JoinRoomPage.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+import { connect } from "react-redux";
+import JoinRoomTitle from "./JoinRoomTitle";
+import JoinRoomContent from "./JoinRoomContent";
+
+import "./JoinRoomPage.css";
+import { setIsRoomHost } from "../../store/actions";
+
+const JoinRoomPage = (props) => {
+  const { setIsRoomHostAction, isRoomHost } = props;
+
+  const search = useLocation().search;
+
+  useEffect(() => {
+    const isRoomHost = new URLSearchParams(search).get("host");
+    if (isRoomHost) {
+      setIsRoomHostAction(true);
+    }
+  }, []);
+
+  return (
+    <div className="join_room_page_container">
+      <div className="join_room_page_panel">
+        <JoinRoomTitle isRoomHost={isRoomHost} />
+        <JoinRoomContent />
+      </div>
+    </div>
+  );
+};
+
+const mapStoreStateToProps = (state) => {
+  return {
+    ...state,
+  };
+};

```

Comment:
Connection parameters: The mapStoreStateToProps function has an unnecessary return statement error. You can simply return the state object without extending it, since it already contains all the state properties.